packages/ibm-products/src/components/Card/Card.tsx

```diff
@@ -252,7 +258,7 @@ export const Card = forwardRef(
             [`${blockClass}__productive`]: productive,
             [`${blockClass}__clickable`]: clickable,
             [`${blockClass}__media-left`]: mediaPosition === 'left',
-            [`${blockClass}--has-slug`]: !!slug,
+            [`${blockClass}--has-slug`]: !!aiLabel || !!slug,
```

Comment:
I think we discussed replacing the classnames too since these are exposed to the consumers.

> Double check if _any styles reference the word slug_ and replace with aiLabel

```suggestion
            [`${blockClass}--has-aiLabel`]: !!aiLabel || !!slug,
```

Reply:
okay got it